coordinator/src/services/translation/request/dispense/dispense-claim.ts

```diff
@@ -68,9 +67,11 @@ export async function convertDispenseClaim(
     dispenseClaim.coverage = new hl7V3.Coverage(chargeExempt)
   }
 
-  //TODO - find an alternative source for this
-  //const hl7PriorPrescriptionReleaseEventRef = createPriorPrescriptionReleaseEventRef(messageHeader)
-  //dispenseClaim.sequelTo = new hl7V3.SequelTo(hl7PriorPrescriptionReleaseEventRef)
+  //This is mandatory but unused by BSA, so just populate it with our own message ID.
+  const hl7PriorPrescriptionReleaseEventRef = new hl7V3.PriorPrescriptionReleaseEventRef(
+    new hl7V3.GlobalIdentifier(messageId)
+  )
+  dispenseClaim.sequelTo = new hl7V3.SequelTo(hl7PriorPrescriptionReleaseEventRef)
```

Comment:
Maybe we could make this more obvious that it's a placeholder, like we have in the past 999**